src/modules/manage.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-alert */
+/* eslint-disable no-lonely-if */
```

Comment:
- [ ] Rather than disable the Linter strive to fix them as this makes you more accustomed to the style guide of the tech.